### PR TITLE
Adds appropriate href to contact section

### LINKED ISSUE #40: Add appropriate href in contact
- Add `tel:` and `mailto:` hrefs for contact section.

### PR DESCRIPTION
**Pull request by rajkumaar23**
Fixes #40

### CODE REVIEW

File: src/containers/contact/Contact.js [hunk 1]
```diff
@@ -1,27 +1,31 @@
...
-        </div>
-        <div className="contact-image-div">
-          <img alt="Saad Working" src={require("../../assests/images/contactMail.png")}></img>
+                    <div className="contact-text-div">
+                        <h1 className="contact-detail">
+                            <a href={"tel:" + contactInfo.number}>{contactInfo.number}</a>
+                        </h1>
+                        <h1 className="contact-detail-email">
+                            <a href={"mailto:" + contactInfo.email_address}>{contactInfo.email_address}</a>
+                        </h1>
```
[the remark this answers is not in the document]
Comment (rajkumaar23) [reply src/containers/contact/Contact.js]: Sure.

**Comment by rajkumaar23** [comment]
Please find the screenshot below : 
image

**Comment by saadpasta** [comment]
> Please find the screenshot below :
> image

Please remove the underline and please change the default anchor hover effect to something different like add shadow

**Comment by rajkumaar23** [comment]
@saadpasta Is this fine ?
dev

**Comment by kartik918** [comment]
@rajkumaar23 we already have a mailto function on the Gmail logo in social icon tray. So making the mail address clickable seems like redundant code. It's aesthetic rather than useful. 

Still if you want to improve on it, add hover transitions that blend with the rest of the page.

**Comment by rajkumaar23** [comment]
@kartik918 I don't think that's redundant. I guess we shouldn't assume everyone has a primary `Gmail` account. In that case, this might help.

File: src/containers/contact/Contact.js [hunk 2]
```diff
@@ -1,27 +1,29 @@
...
+                        <a className="contact-detail-email"
+                           href={"mailto:" + contactInfo.email_address}>{contactInfo.email_address}</a>
+                        <br/><br/>
+                        <SocialMedia/>
+                    </div>
+                </div>
+                <div className="contact-image-div">
+                    <img alt="Saad Working" src={require("../../assests/images/contactMail.png")}></img>
+                </div>
+            </div>
```
Comment (kartik918) [inline src/containers/contact/Contact.js]: Try to preserve code indentation to avoid unnecessary spacing getting committed as new changes.

**Comment by kartik918** [comment]
> 
> 
> @kartik918 I don't think that's redundant. I guess we shouldn't assume everyone has a **primary Gmail** account. In that case, this might help.

mailto: is independent of the mail platform you use, but I get where you're coming from.
Maybe instead of a "G" logo, use mail icon @saadpasta ?

File: src/containers/contact/Contact.js (hunk 2 above)
Comment (rajkumaar23) [reply src/containers/contact/Contact.js]: I reformatted using PhpStorm. Now it **looks** clean. If you still want me to revert, I would.
image

**Comment by rajkumaar23** [comment]
> > @kartik918 I don't think that's redundant. I guess we shouldn't assume everyone has a **primary Gmail** account. In that case, this might help.
> 
> mailto: is independent of the mail platform you use, but I get where you're coming from.
> Maybe instead of a "G" logo, use mail icon @saadpasta ?

Why not replace with some other social media as in Instagram or Twitter ?

**Comment by kartik918** [comment]
> Why not replace it with some other social media as in Instagram or Twitter ?

Well it is upto the person using it to customize it as per their needs.

As for the PR, add some delayed transitions for on hover to make them look smoother.
Preferably, ``transition: all 0.3s;``

File: src/containers/contact/Contact.js (hunk 2 above)
Comment (kartik918) [reply src/containers/contact/Contact.js]: You could've made a seperate commit for reformatting all code to one format of indents but it's okay now.
Just keep in mind for future contributions.
Other than that, great PR!

**Comment by rajkumaar23** [comment]
> Well it is upto the person using it to customize it as per their needs.
> 
> As for the PR, add some delayed transitions for on hover to make them look smoother.
> Preferably, `transition: all 0.3s;`

Done :+1:

**Comment by saadpasta** [comment]
@rajkumaar23 Great Work. Please try to preserve code indentation to avoid unnecessary spacing as said by @kartik918 for the future other then that awesome work ❤️

**Comment by saadpasta** [comment]
@all contributors please add @rajkumaar23 for code

**Comment by saadpasta** [comment]
@allcontributors please add @rajkumaar23 for code

**Comment by rajkumaar23** [comment]
> @allcontributors please add @rajkumaar23 for code

Thanks :))

**Pull request merged**